Java/core/src/main/java/com/amazon/randomcutforest/tree/CompactNodeViewDouble.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.randomcutforest.tree;
+
+import java.util.Arrays;
+import java.util.Set;
+
+public class CompactNodeViewDouble implements INodeView<double[]> {
+    final CompactRandomCutTreeDouble treeRef;
+    short currentNodeOffset;
+
+    public CompactNodeViewDouble(short nodeId, CompactRandomCutTreeDouble treeRef) {
```

Comment:
minor: I would change this signature to:
```
public CompactNodeViewDouble(CompactRandomCutTreeDouble tree, short initialNodeIndex)
```

Reply:
fixed